crates/sparrow-catalog/catalog/yearly.toml

```diff
@@ -66,8 +66,8 @@ time,key,n
 '''
 output_csv = '''
 time,key,n
-1996-12-19T16:00:00.000000000,Ben,2
 1996-12-19T16:00:00.000000000,Ryan,3
-1997-12-20T16:01:00.000000000,Ben,9
+1996-12-19T16:00:00.000000000,Ben,2
 1997-12-21T16:00:00.000000000,Ryan,8
+1997-12-20T16:01:00.000000000,Ben,9
```

Comment:
Many of these results were reordered -- I assume that is because the `_key_hash` changed (which we drop) and reordered the results according to the hash?

Reply:
I believe so, yes.